expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java

```diff
@@ -277,6 +253,41 @@ void shouldNotFailWhenInputContainsAffiliationsThatAreIncomplete() {
         assertDoesNotThrow(() -> ExpandedResource.fromPublication(uriRetriever, publication));
     }
 
+
+    private void assertExplicitFieldsFromFraming(ObjectNode framedResultNode) {
+        var node = framedResultNode.at("/topLevelOrganization");
+        StreamSupport.stream(node.spliterator(), false)
+            .flatMap(ExpandedResourceTest::getaVoid)
+            .forEach(ExpandedResourceTest::assertAllFieldsAreValid);
+    }
+
+    private static Stream<String> getaVoid(JsonNode topOrg) {
```

Comment:
Its hard to udnerstand this method, and the name does not help. I read it as "get a void" but that does not make any sense

Reply:
Auto-refactor, alas. I thought I had renamed it; I was mistaken